e2e/pages/app/app.module.ts

```diff
@@ -2,28 +2,73 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
```

Comment:
When i pull down a fresh copy, run npm install and then try to run grunt documentation:build i get a load of errors, looks like some things are either defined as `any` or not given a type:

![image](https://user-images.githubusercontent.com/20795331/28020300-c5ed2724-657b-11e7-99b9-55b66d26669d.png)